extension/app/html/background.html

```diff
@@ -23,6 +21,7 @@
 
 		<main>Loading...</main>
 
+		<!-- a workaround for bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1803984 -->
 		<script type = 'module' src = '../js/background/background.js'></script>
```

Comment:
```suggestion
		<!-- a workaround for bug: https://bugzilla.mozilla.org/show_bug.cgi?id=1803984 -->
		<script type = 'module'>import '../js/background/background.js'</script>
```
If background.html doesn't suffer from the same bug and we have verified everything works in FF, then we should instead do:
```suggestion
		<script type = 'module' src = '../js/background/background.js'></script>
```